RDKEMW-6213 : subtec unit tests update

krithikasvraman opened the pull request:
Reason for change: github copilot used to check and add/update tests 
Test Procedure: verify unit tests are passing
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-6213 : subtec unit tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: verify unit tests are passing
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 5 files pending identification.
* Protex Server Path: /home/blackduck/github/subtec-app/48/components/generic/subtec-app

* Commit: c2bf7e4c9c1a9a1ec05b5d6877a5ce7754670432

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/79bf1338232b577b1e472452d040242c)'

rdkcmf-jenkins commented:
b'## **WARNING:** A Blackduck scan failure has been waived

A prior failure has been upvoted
* Upvote reason: OK

* Commit: c2bf7e4c9c1a9a1ec05b5d6877a5ce7754670432
'

krithikasvraman committed:
Merge branch 'develop' into feature/RDKEMW-6213

krithikasvraman committed:
Merge branch 'develop' into feature/RDKEMW-6213

krithikasvraman committed:
RDKEMW-6213 : subtec unit tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: verify unit tests are passing
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 35 files pending identification.
* Protex Server Path: /home/blackduck/github/subtec-app/48/components/generic/subtec-app

* Commit: d1ac864e2481cba407913b01ef83261fa005d55e

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/08ea8f74db77995b232011daabcb6fdc)'

krithikasvraman committed:
Merge branch 'develop' into feature/RDKEMW-6213

krithikasvraman committed:
RDKEMW-6213 : subtec unit tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: verify unit tests are passing
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

rdkcmf-jenkins commented:
b'## Copyright scan failure
 **Commit:** 20fa66c0d5a8a6e06f39588751d76a0dec0974fc
**Report detail:**  https://gist.github.com/rdkcmf-jenkins/d895370cf0e5a8dfe9b64b6085aabe96'

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 30 files pending identification.
* Protex Server Path: /home/blackduck/github/subtec-app/48/components/generic/subtec-app

* Commit: 20fa66c0d5a8a6e06f39588751d76a0dec0974fc

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/22254c2e2faff58df1e160fe3af8a2e2)'

krithikasvraman committed:
Merge branch 'develop' into feature/RDKEMW-6213

krithikasvraman committed:
RDKEMW-6213 : subtec unit tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: verify unit tests are passing
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

rdkcmf-jenkins commented:
b'## **Blackduck scan failure details**

**Summary**: 0 **violation**s, 0 files pending approval, 33 files pending identification.
* Protex Server Path: /home/blackduck/github/subtec-app/48/components/generic/subtec-app

* Commit: dc7e76dd031d5d237e9613d414fb2d7c27a7b5ed

 **Report detail:** [gist](https://gist.github.com/rdkcmf-jenkins/bae418a27d91a4cac1f8f7fc4330a924)'

rdkcmf-jenkins commented:
b'## **WARNING:** A Blackduck scan failure has been waived

A prior failure has been upvoted
* Upvote reason: ok

* Commit: dc7e76dd031d5d237e9613d414fb2d7c27a7b5ed
'

krithikasvraman committed:
Merge branch 'develop' into feature/RDKEMW-6213

krithikasvraman committed:
RDKEMW-6213 : L1, L2 tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: Run unit tests
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-6213 : L1, L2 tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: Run unit tests
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-6213 : L1, L2 tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: Run unit tests
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-6213 : L1, L2 tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: Run unit tests
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-6213 : L1, L2 tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: Run unit tests
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
Merge branch 'develop' into feature/RDKEMW-6213

krithikasvraman committed:
RDKEMW-6213 : L1, L2 tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: Run unit tests
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman committed:
RDKEMW-6213 : L1, L2 tests update

Reason for change: github copilot used to check and add/update tests
Test Procedure: Run unit tests
Risks: Low
Signed-off-by:krithika_venkataraman@comcast.com

krithikasvraman commented:
This PR is abandoned as this was huge, and it has been split into smaller PRs and merged to develop.
